src/songMetadata/songMetadata.ts

```diff
@@ -0,0 +1,149 @@
+import {
+    SearchReleasesFields,
+    musicBrainzClient,
+    SearchReleasesCriteria,
+    coverArtArchiveClient,
+    Recording,
+} from './musicBrainzModule';
+
+export interface ISongMetadataMatchedResult {
+    title: string;
+    artists: string;
```

Comment:
Split this to "artists array" and "artists display name"

Reply:
Waiting for this.